indexer/search_string_utils.cpp

```diff
@@ -330,13 +330,13 @@ class StreetsSynonymsHolder
     char const * affics[] =
     {
       // Russian - Русский
-      "улица", "ул",
+      "улица", "ул", "проспект",
```

Comment:
Some time ago I purposely removed many rare synonyms, because they break normal flow.
Now to respect better ranking I returned only major street synonyms (проспект, avenue).